src/components/map/Map.js

```diff
@@ -20,18 +20,18 @@ class Map extends Component {
   constructor() {
     super();
 
+    navigator.geolocation.getCurrentPosition((position) => {
```

Comment:
Not sure about having this in the constructor method, but not sure it's wrong either. Why did you change it?

Reply:
Because the linter was complaining about applying `setState` in the `componentDidMount` method